apps/etterlatte-saksbehandling-ui/client/src/components/behandling/soeknadsoversikt/familieforhold/barnepensjon/FamilieforholdBarnepensjon.tsx

```diff
@@ -0,0 +1,61 @@
+import styled from 'styled-components'
+import { GjenlevendeForelder } from './../personer/GjenlevendeForelder'
+import { Barn } from '../personer/Barn'
+import { Border, DashedBorder } from '../../styled'
+import { SoeskenListe } from './../personer/Soesken'
+import { GyldigFramsattType, IDetaljertBehandling, IGyldighetproving } from '~shared/types/IDetaljertBehandling'
+import { VurderingsResultat } from '~shared/types/VurderingsResultat'
+import { Foreldre } from '~components/behandling/soeknadsoversikt/familieforhold/personer/Foreldre'
+
+export interface PropsFamilieforhold {
+  behandling: IDetaljertBehandling
+}
+
+export const FamilieforholdBarnepensjon: React.FC<PropsFamilieforhold> = ({ behandling }) => {
+  const innsenderErGjenlevende =
+    behandling.gyldighetsprøving?.vurderinger.find(
+      (g: IGyldighetproving) => g.navn === GyldigFramsattType.INNSENDER_ER_FORELDER
+    )?.resultat === VurderingsResultat.OPPFYLT
+
+  if (behandling.familieforhold == null || behandling.søker == null) {
+    return <div style={{ color: 'red' }}>Familieforhold kan ikke hentes ut</div>
```

Comment:
Har ikke ds en konstant for denne fargen? Evt en komponent som er vanlig å bruke.